src/PrestaShopBundle/Resources/config/routing/admin/configure/advanced_parameters/shops.yml

```diff
@@ -0,0 +1,7 @@
+admin_shops_search:
+  path: /search/{searchTerm}
+  methods: [GET]
+  defaults:
+    _controller: 'PrestaShopBundle:Admin\Configure\AdvancedParameters\Shop:search'
```

Comment:
I think you need a `_legacy_controller` for some of the stuff to work 🤔 
- tabs
- authorization

ping @jolelievre @zuk3975

Reply:
I was unsure if it was needed.... since there was no corresponding "search" method or similar in the legacy AdminShopController...